tests/ethpm/test_package.py

```diff
@@ -1,34 +1,49 @@
 import pytest
 from ethpm.package import Package
 from ethpm.exceptions import ValidationError
+from web3 import (
+    Web3,
+    HTTPProvider,
+)
 
 
 def test_ethpm_exists():
     assert Package
 
 
 @pytest.fixture()
-def valid_package():
+def valid_package_id():
     return "./tests/ethpm/validLockfile.json"
 
 
 @pytest.fixture()
-def invalid_package():
+def invalid_package_id():
     return "./tests/ethpm/invalidLockfile.json"
 
 
-def test_ethpm_instantiates_with_valid_package(valid_package):
-    current_package = Package(valid_package)
-    assert current_package.package_identifier == valid_package
+@pytest.fixture()
+def web3():
+    return Web3(HTTPProvider('http://localhost:8545'))
```

Comment:
You *should* switch to use `web3.providers.eth_tester.EthereumTesterProvider` here.